config/locales/en.yml

```diff
@@ -1857,16 +1857,21 @@ en:
             <li>money spent on fundraisers including raffles, bingo; or</li>
             <li>items purchased for a fundraiser.</li></ul>
           learn_more_title: What does this mean?
-          title: Did you make charitable contributions (cash and/or non-cash) to a qualifying organization in 2023?
+          title:
+            one: Did you make charitable contributions (cash and/or non-cash) to a qualifying organization in %{tax_year}?
+            other: Did you or your spouse make charitable contributions (cash and/or non-cash) to a qualifying organization in %{tax_year}?
```

Comment:
Maybe consider calling these `not_mfj`/`mfj` to be more descriptive, but it's not a big deal

Reply:
@mpidcock this count feature is something that is built into to [I18nt](https://www.i18next.com/translation-function/plurals) so I wouldn't be able to change it into  `not_mfj`/`mfj` but I could change it into something like `not_mfj_one` and `mfj_other` although I prefer to keep it `one`/`other` since that has been the pattern in GYR

Reply:
ah, I was worried it might be something like that but couldn't figure out where the names come from. thanks, and I agree that keeping it is good to match GYR